src/configs/repositories/configRepository.ts

```diff
@@ -177,10 +153,22 @@ export class ConfigRepository {
     }));
 
     await this.drizzle.transaction(async (tx) => {
-      await tx.insert(configs).values(configData).execute();
+      await tx
+        .insert(configs)
+        .values({ ...configData, isLatest: true })
+        .execute();
+
       if (dbRefs.length > 0) {
         await tx.insert(configsRefs).values(dbRefs).execute();
       }
+
+      if (config.version !== 1) {
```

Comment:
Maybe add a comment explaining that this logic is meant so that the last version which was marked `latest` is not anymore.

Reply:
explained